openedx/core/djangoapps/credit/services.py

```diff
@@ -48,7 +48,7 @@ def is_credit_course(self, course_key_or_id):
 
         return is_credit_course(course_key)
 
-    def get_credit_state(self, user_id, course_key_or_id, return_course_name=False):
+    def get_credit_state(self, user_id, course_key_or_id, return_course_info=False):
```

Comment:
Now I'm worried that renaming the keyword param here is going to break stuff we don't know about. Are we aware of all clients of the credit service and this get_credit_state function?